http-core/src/main/scala/org/apache/pekko/http/impl/engine/http2/Http2Blueprint.scala

```diff
@@ -127,6 +127,7 @@ private[http] object Http2Blueprint {
       serverDemux(settings.http2Settings, initialDemuxerSettings, upgraded) atop
       FrameLogger.logFramesIfEnabled(settings.http2Settings.logFrames) atop // enable for debugging
       hpackCoding(masterHttpHeaderParser, settings.parserSettings) atop
+      rapidResetMitigation(settings.http2Settings) atop
```

Comment:
Better with an option to not include this stage into flow, if the endpoint is not public but internal.

Reply:
makes sense - there are 3 new configs - one is an interval - what about me updating the code so that the interval is 0 then the feature is just disabled?

Reply:
made a change